ml/src/main/scala/ai/h2o/sparkling/ml/params/H2OMOJOAlgoSharedParams.scala

```diff
@@ -47,21 +47,18 @@ trait H2OMOJOModelParams extends Params with Logging {
   protected final val namedMojoOutputColumns: Param[Boolean] = new BooleanParam(this, "namedMojoOutputColumns", "Mojo Output is not stored" +
     " in the array but in the properly named columns")
 
-  protected final val modelDetails: NullableStringParam = new NullableStringParam(this, "modelDetails", "Raw details of this model.")
-
   //
   //
   // Default values
   //
   setDefault(
-    featuresCols -> Array.empty[String],
     predictionCol -> H2OMOJOSettings.default.predictionCol,
-    convertUnknownCategoricalLevelsToNa -> H2OMOJOSettings.default.convertUnknownCategoricalLevelsToNa,
-    convertInvalidNumbersToNa -> H2OMOJOSettings.default.convertInvalidNumbersToNa,
-    namedMojoOutputColumns -> H2OMOJOSettings.default.namedMojoOutputColumns,
     detailedPredictionCol -> H2OMOJOSettings.default.detailedPredictionCol,
     withDetailedPredictionCol -> H2OMOJOSettings.default.withDetailedPredictionCol,
-    modelDetails -> null
+    featuresCols -> Array.empty[String],
+    convertUnknownCategoricalLevelsToNa -> H2OMOJOSettings.default.convertUnknownCategoricalLevelsToNa,
+    convertInvalidNumbersToNa -> H2OMOJOSettings.default.convertInvalidNumbersToNa,
+    namedMojoOutputColumns -> H2OMOJOSettings.default.namedMojoOutputColumns
```

Comment:
Is there any relationship between `namedMojoOutputColumns` and `predictionCol`and `predictionDetailsCol`?

Reply:
no

Reply:
May i know what's their purpose?

Reply:
predictionCol and predictionDetailsCols are obvious I guess.

for `namedMojoOutputColumns` -> used only on MojoPipeline, when enabled, the outputs are named by the label cols, instead of returning regular array. We will get rid of this option after we have simple output in predictionCol.
https://github.com/h2oai/sparkling-water/blob/610332aa421f9b20f38bf9a582d915cc5aa689b8/ml/src/main/scala/org/apache/spark/ml/h2o/models/H2OMOJOPipelineModel.scala#L128

Though changing semantic of these is not goal of this change

Reply:
Thanks for the explanation!

Reply:
no worries